GameData/RealismOverhaul/RO_SuggestedMods/FASA/RO_FASA_ApolloCSM.cfg

```diff
@@ -137,29 +137,48 @@
 	@manufacturer = North American
 	@description = The three main recovery parachutes for the Apollo Command Module.
 	@mass = 0.1684
-	@MODULE[RealChuteModule]
-	{
-		@caseMass = 0.0931
-		@mustGoDown = true
-		@spareChutes = 0
-		@PARACHUTE
-		{
-			@preDeployedDiameter = 3.17
-			@deployedDiameter = 60.65
-			@minIsPressure = false
-			%minDeployment = 3200
-			@deploymentAlt = 700
-			@preDeploymentSpeed = 1
-			@deploymentSpeed = 1
-			@parachuteName = Canopy
-			@capName = Cap
-		}
+
+	@MODULE[ModuleParachute]:NEEDS[!RealChute]
+	{
+		@autoCutSpeed = 0.5
+		@stowedDrag = 0
+		@semiDeployedDrag = 2
+		@fullyDeployedDrag = 3
+		@minAirPressureToOpen = 0.02
+		@clampMinAirPressure = 0.02
+		@deployAltitude = 3350
+		@deploymentSpeed = 2.5
+		@semiDeploymentSpeed = 5.0
+		@chuteMaxTemp = 473.15
+		@chuteThermalMassPerArea = 0.09
 	}
-	@MODULE[ModuleParachute]
-	{
-		@semiDeployedDrag = 4
-		@fullyDeployedDrag = 800
-		@deployAltitude = 1000
+
+	!MODULE[ModuleParachute]:NEEDS[RealChute] {}
+	MODULE:NEEDS[RealChute]
+	{
+		name = RealChuteModule
+		caseMass = 0.1106
+		mustGoDown = True
+		spareChutes = 0
+		cutSpeed = 0.5
+
+		PARACHUTE
+		{
+			parachuteName = Canopy
+			capName = Cap
+			preDeploymentAnimation = Apollo_Chute_Semi
+			deploymentAnimation = Apollo_Chute_Full
```

Comment:
This sort of thing, with references to animations, really seems like it needs to be in RO_FASA instead of RO core.

Reply:
I couldn't get it to work any other way. Something is weird about the config in FASA I think.

Reply:
I was under an apparently-incorrect assumption that RealChute compatibility in general was done either by RC or by the original part mod.  Neither, at least in the case of FASA, and there's a few other full RC configs in RO core.

https://github.com/KSP-RO/FASA-RO/blob/master/GameData/FASA/Apollo/ApolloCSM/FASA_Apollo_CM_parachutes.cfg is the original config.  I don't see some of the fields you adjust above in ModuleParachute there, so @ would do nothing unless something -else- is poking them first.  I'm -really- not sure that "2" and "3" are reasonable values for [semi]deployed drag, and they conflict with what RO set before.  There's likely not a lot of testing in RO *without* RealChute.